test/e2e_tests/pageManager/webapp/pages/conversation.page.ts

```diff
@@ -52,7 +53,7 @@ export class ConversationPage {
   readonly callButton: Locator;
   readonly conversationInfoButton: Locator;
   readonly pingButton: Locator;
-  readonly messages: Locator;
+  readonly messages: Locator; // Only contains message items which have been sent successfully
```

Comment:
might be useful to use a jsdoc style comment instead


```suggestion
  /**
    * Only contains message items which have been sent successfully
  */
  readonly messages: Locator; // Only contains message items which have been sent successfully
```

Reply:
Good point, then it will be shown when using the property somewhere 👍